src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeValidator.java

```diff
@@ -47,7 +47,8 @@
         ImmutableMap.of(
             Classification.EXCEPTION, emptyList(),
             Classification.NEW_APPLICATION, emptyList(),
-            Classification.SUPPLEMENTARY_EVIDENCE, asList(InputDocumentType.FORM, InputDocumentType.SSCS1)
+            Classification.SUPPLEMENTARY_EVIDENCE, asList(InputDocumentType.FORM, InputDocumentType.SSCS1),
+            Classification.SUPPLEMENTARY_EVIDENCE_WITH_OCR, emptyList()
```

Comment:
is it ok for supplementary evidence with OCR to have docs of type `FORM`?

Reply:
the document containing ocrData is usually of type "FORM"